src/main/java/com/endava/internship/collections/Node.java

```diff
@@ -0,0 +1,20 @@
+package com.endava.internship.collections;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Setter @Getter
+public class Node {
+    private final Student key;
+    private Integer value;
+
+    private Node left = null;
```

Comment:
Instance variables are null by default, not need to write "=null"

Reply:
Fixed.